application-forum-ui/src/main/resources/ForumCode/AnswerSheet.xml

```diff
@@ -53,6 +53,21 @@
         &lt;/dl&gt; 
       &lt;/div&gt;
     {{/html}}
+    #elseif ($request.commentEditForm == 'true')
+    {{html}}
+      #set ($commentEditForm = "#template('commentsinline.vm')")
+      #if (!$commentEditForm.contains('&lt;textarea/&gt;'))
+        #set ($discard = $doc.use('XWiki.XWikiComments', $request.number))
+        #set ($commentEditField = $doc.display('comment', 'edit'))
+        #set ($insertPosition = $commentEditForm.indexOf('&lt;!-- loaded by javascript'))
+        #if ($insertPosition != -1)
+          #set ($commentEditForm = $commentEditForm.substring(0, $insertPosition) + $commentEditField + $commentEditForm.substring($insertPosition))
+        #end
+      #end
+      #set ($commentEditForm = $stringtool.remove($commentEditForm, '{{html clean="false" wiki="false"}}'))
+      #set ($commentEditForm = $stringtool.remove($commentEditForm, '{{/html}}'))
```

Comment:
Where does this ``{{html}}`` macro comes from? ``commentsinline.vm`` or ``$doc.display('comment', 'edit')``? If it's the later, then why remove it when ``$doc.display('comment', 'edit')`` is not called? And, wouldn't it be more clear to remove these from the ``$commentEditField`` instead?